site/docs/pages/wallet/wallet-dropdown-link.mdx

```diff
@@ -0,0 +1,195 @@
+  ConnectWallet,
+  Wallet,
+  WalletDropdown, 
+  WalletDropdownLink, // [!code focus]
+  WalletDropdownDisconnect,
+} from '@coinbase/onchainkit/wallet';
+
+export function WalletComponents() {
+  return (
+    <div className="flex justify-end">
+      <Wallet>
+        <ConnectWallet>
+          <Avatar className="h-6 w-6" />
+          <Name />
+        </ConnectWallet>
+        <WalletDropdown>
+          <Identity className="px-4 pt-3 pb-2" hasCopyAddressOnClick>
+            <Avatar />
+            <Name />
+            <Address className={color.foregroundMuted} />
+            <EthBalance />
+          </Identity>
+          <WalletDropdownLink icon="wallet" href="https://wallet.coinbase.com"> // [!code focus]
+            Wallet // [!code focus]
+          </WalletDropdownLink> // [!code focus]
+          <WalletDropdownDisconnect />
+        </WalletDropdown>
+      </Wallet>
+    </div>
+  );
+}      
+```
+
+<WalletComponents>
+  <Wallet>
+    <ConnectWallet>
+      <Avatar className="h-6 w-6" />
+      <Name />
+    </ConnectWallet>
+    <WalletDropdown>
+      <Identity className="px-4 pt-3 pb-2" hasCopyAddressOnClick>
+        <Avatar />
+        <Name />
+        <Address className={color.foregroundMuted} />
+        <EthBalance />
+      </Identity>
+      <WalletDropdownLink icon="wallet" href="https://wallet.coinbase.com">
+        Wallet
+      </WalletDropdownLink>
+      <WalletDropdownDisconnect />
+    </WalletDropdown>
+  </Wallet>
+</WalletComponents>
+
+### Custom link
+Add a custom link to the wallet dropdown menu, allowing users to navigate to external resources directly from the wallet interface.
+
+```tsx
+// omitted for brevity
+<WalletDropdownLink // [!code focus] 
+  icon={BaseIcon} // [!code focus] 
+  href="https://www.base.org/" // [!code focus] 
+  rel="noopener noreferrer" // [!code focus] 
+> // [!code focus]
+  Base.org // [!code focus]
+</WalletDropdownLink> // [!code focus]
+```
+
+<WalletComponents>
+  <Wallet>
+    <ConnectWallet>
+      <Avatar className="h-6 w-6" />
+      <Name />
+    </ConnectWallet>
+    <WalletDropdown>
+      <Identity className="px-4 pt-3 pb-2" hasCopyAddressOnClick>
+        <Avatar />
+        <Name />
+        <Address className={color.foregroundMuted} />
+        <EthBalance />
+      </Identity>
+      <WalletDropdownLink 
+        icon={walletDropdownLinkCustomBaseIconSvg} 
+        href="https://www.base.org/" 
+        rel="noopener noreferrer" 
+      >
+        Base.org
+      </WalletDropdownLink>
+      <WalletDropdownDisconnect />
+    </WalletDropdown>
+  </Wallet>
+</WalletComponents>
+
+### Custom children components
+Accepts React children, enabling the use of custom elements, styled text, icons, and complex components.
+This allows for diverse customizations, including complex layouts and conditional rendering based on your app's state.
+
+```tsx
+// omitted for brevity
+<WalletDropdownLink icon="wallet" href="https://wallet.coinbase.com"> // [!code focus]
```

Comment:
Follow up for next PR:
- teach how to use custom icons react components. And not just the default `wallet` one.